relay-general/src/protocol/contexts/app.rs

```diff
@@ -39,6 +39,9 @@ pub struct AppContext {
     /// A flag indicating whether the app is in foreground or not. An app is in foreground when it's visible to the user.
     pub in_foreground: Annotated<bool>,
 
+    /// The name of the currently visible view.
+    pub view_name: Annotated<String>,
```

Comment:
I guess this should be an Array of Strings though, Imagine foldable devices that have screens A and B shown/visible at the very same time.
What do you think?

Reply:
Is this something that was requested by a user? Would initialy keep it simple, but don't have a strong opinion here.

Reply:
Sound reasonable to use an array. For iPads would be the same thing. We may have two windows open and we dont have a concept of foreground window because the user can interact with both at the same time.

Reply:
On RN multi-window apps are not common, but it's possible.

Array of the names makes sense to me.

Reply:
@denrase vox populi 🗡️